codalab/apps/web/models.py

```diff
@@ -351,7 +359,7 @@ def get_chahub_data(self):
             submitted_at__gt=now() - datetime.timedelta(days=30)
         ).exists()
 
-        return [{
+        return [self.clean_private_data({
```

Comment:
should be sending creator_id, too, not just username. probably some other stuff in here that may have broken w/ the adjustments to the endpoint on chahub, bbuuut I think the bulk (and important stuff) should still be working...

Reply:
```
'creator_id': self.uploaded_by.id,
```

am I missing something or that's already being sent? the username is _also_ attached but maybe not super useful?

I tested sending a dataset again, appears to still work with one slight hiccup I'll PM you about

Reply:
This is a competition, not dataset, I don't see creator_id here.

Reply:
my mistake. fixed.

Reply:
uh, sorry, created_by _is_ a string, `creator_id` is an additional field, that should be a pk, and the one we whitelist.

Reply:
To clarify, for Competitions I'll whitelist `creator_id` and send along the following fields to Chahub:

```            "created_by": str(self.creator),
            "creator_id": self.creator.pk,```

(confirmed from v2 this is how you're doing it)